.github/workflows/build.yml

```diff
@@ -214,6 +212,12 @@ jobs:
             add_to_matrix "ppc853x-5.2"
           fi
 
+          # Add SRM 1.3 builds
+          if [ "$add_srm13_builds" == "true" ]; then
+            add_to_matrix "aarch64-1.3"
+            add_to_matrix "armv7-1.3"
+          fi
```

Comment:
Is the intent to add srm-1.3 to the default build matrix?  Just for my understanding...
Also something may be missing around lines 14x for this to get trigerred?

Reply:
So far we do not build packages for SRM (1.2 and 1.3) by default, but we can build and publish those with manual build workflow (as we do for DSM 5.2).